applications/application-health.adoc

```diff
@@ -29,5 +29,5 @@ include::modules/odc-monitoring-health-checks.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* For details on switching to the *Developer* perspective in the web console, see xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[About *Developer* perspective].
+* For details on switching to the *Developer* perspective in the web console, see xref:../web_console/about-developer-perspective.adoc#about-developer-perspective_web-console-overview[About *Developer* perspective].
```

Comment:
I could be wrong, but part of the build failure could be related to the inclusion of the context  __web-console-overview in the path. Does it resolve when the ID is updated to about-developer-perspective alone?